VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -171,19 +172,9 @@
                         tools:targetApi="17" />
 
                     <Switch
+                        style="@style/edX.Widget.Switch"
```

Comment:
While the attribute was added to the public API in API 21, it _is_ actually used in the AOSP widget since the start I think, and should actually work if used even though it produces a lint error. However, I guess it's best to not reply on undocumented API :wink: